fluent-plugin-google-cloud.gemspec

```diff
@@ -19,7 +19,7 @@ eos
   gem.test_files    = gem.files.grep(/^(test)/)
   gem.require_paths = ['lib']
 
-  gem.add_runtime_dependency 'fluentd', '1.6.3'
+  gem.add_runtime_dependency 'fluentd', '1.9.0'
```

Comment:
Your PR also bumps the fluentd version dependency, and I'm not sure we're prepared to do that yet.

Reply:
ok, let me try reverting this change

Reply:
hmm i guess MessagePackFactory.msgpack_packer is not defined in 1.6.3. please go ahead and close this PR if you can't bump the fluentd version yet. i'll apply this patch locally.